.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: A short description of the new feature
+labels: enhancement
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**WordPress version**
+Add the WordPress version you are currently using.
+
+**WooCommerce version**
+Add the WooCommerce version you are currently using.
+
+**Module version**
+Add the Smaily For WooCommerce module version you are currently using.
```

Comment:
I have used the same basic template across plugins but maybe we should omit versioning from feature request templates. For a bug report template, it would make sense but for a feature request template you wouldn't probably use it and the feature will be added to the latest version of the plugin anyway.

Maybe WordPress and WooCommerce version should be kept as it allows us to see the major version when it matters.

Reply:
I would remove all versioning information from the feature request template, as it would it more clear, and we would need to build the feature to be compatible for all supported WordPress and WooCommerce versions anyway.

Reply:
Removed the whole versioning block https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/57/commits/282dbb4f5573a58624fb92f863ee7581c91f83f0